\chapter{Suppression of transposable elements in Leukemic stem cells \cite{Colombo2017}}
\label{cha:research_topic_4}

\section{Introduction}
Transposable elements (TEs) were first discovered by Barbara McClintock in the 1940s while studying colouring patterns in maize kernels. Later, it was discovered that large parts of both eukaryotic and prokaryotic genomes consist of transposable elements. Alternatively called `jumping genes' or transposons because of their ability to move (or jump) from one location in the genome to another, TEs are particularly abundant in plant genomes, with 80\% of wheat genome comprising of TEs \cite{brenchly12}. While initially dismissed as `junk' DNA, there is increasing debate on the role of TEs in evolution \cite{chuong16,lisch13}. TEs are also referred to as repeat elements because they are present in multiple copies across the genome. 

TEs are largely categorized by the mechanism of their movement in genomes; those that move by a `copy-and-paste' mechanism using an RNA intermediate (class I retrotransposons), and those that use a `cut-and-paste' mechanism with a DNA intermediate (class II transposons). Due to their ability to make duplicate copies, TEs have been implicated in the increase of genome size. Particularly in angiosperm genomes, that vary over 1000-fold in size, the variation in genome size is strongly correlated with TE content \cite{tenaillon10}. 

Retrotransposons are further classified into long terminal repeat (LTR) and non-LTR elements. Endogenous retroviruses (ERV), which are LTRs, resemble retroviruses in their structure and function. Long interspersed nuclear elements (LINE) such as LINE1 are non-LTRs, and autonomous in their ability to retrotranspose, whereas short interspersed nuclear elements (SINE) such as Alu are non-autonomous, and dependent on LINE for retrotransposition. TEs are highly expressed during embryogenesis and play an active role in it \cite{Gerdes16,Ge17}. TEs have also been suggested to have played a positive role in evolution by increasing the potential for advantageous novel genes \cite{Elbarbary16, Sundaram2014,Thompson2016, Lee2015}.

The genomic regions that contain TEs are highly methylated and are silenced by heterochromatin in the somatic cells \cite{Schulz2006, Groh2017}. TE activation has been reported in aging tissues, including in aging stem cells \cite{Wang2011,Sun2014}. TEs have been reported to be expressed in various types of cancers for the past three decades; however, it remains unknown if they are causal or consequential to the development of cancer. In humans, TEs have been mostly considered detrimental because of their inherent mobile nature. Their expression can lead to insertional mutagenesis, chromosomal rearrangements, and genomic instability, potentially contributing to cancer development \cite{Belancio2015, Kemp2015, Mills2007, Luzhna2015}. 

Recent reports revealed a potential beneficial role of TEs in cancer, wherein ERVs were shown to be potential tumour-specific antigens \cite{Mullins2012}. Hypomethylating agents increase the expression of TEs in cancer cells, inducing `viral mimicry' and causing interferon signalling and cancer cell killing \cite{Chiappinelli2015, Roulois2015}. Bidirectional (sense and anti-sense) transcription of many TEs, including ERVs, yields dsRNA \cite{Lehner2002, Yelin2003}. dsRNA sensors then activate potent interferon response pathways, leading to the activation of inflammatory pathways and cell death \cite{Chiappinelli2015, Roulois2015}. These findings suggested that TE expression in cancer cells could play a role in immune-mediated clearance of cancer cells.

Acute myeloid leukaemia (AML), the most common form of acute leukaemia in adults, is characterized by high rates of initial remission with chemotherapy (60 - 70\%), but is also associated with high relapse rates. Nearly two decades ago, it was shown that only a small fraction of AML cells (termed leukemic stem cells or LSCs) were capable of re-initiating the tumour when transplanted into immunodeficient animals \cite{Bonnet1997}. LSCs in AML can be identified based on the expression of cell surface proteins ($CD34^+CD38^{neg}CD99^+TIM3^+$) \cite{Corces2016}. Although the exact role of LSCs in the pathogenesis and relapse of AML is still debated, their presence is associated with resistance to therapy, relapse, and poor prognosis \cite{Reinisch2015}. Thus, targeting LSCs in AML is a major focus of oncologic research, however the lack of understanding of pathways dysregulated in LSCs has hampered progress. We speculated that the resilience of LSCs was mediated by its ability to escape immune mediated clearance. To investigate this, we studied the expression of TEs and its accompanying immune pathways in AML cell fractions.

\section{Materials and Methods}

\subsection{ATACseq data analysis}
ATACseq fastq reads were analyzed from Corces et al \cite{Corces2016}. Patient trios were selected which had ATACseq data from pHSC, LSC and Blast samples. ATACseq reads were aligned against the hg19 reference genome using the \href{https://arxiv.org/abs/1303.3997}{bwa mem algorithm}. Following the alignment, QC was performed for the samples. Only reads aligned to autosomes and sex chromosomes were considered. Mitochondrial reads were discarded.

All samples showed enrichment for transcription start sites (TSSs) sites. Enrichment was computed by comparing total reads falling into a window of 2kb just upstream of promoters to reads in a 5kb window distant from the TSS. An important visualisation tool for ATACseq data is the distribution of fragment sizes. A typical fragment size distribution showed a characteristic wiggle indicating large fraction of short nucleosome-free fragments and a lower fraction of fragments from regions protected by nucleosomes \cite{Buenrostro2013, buenrostro2015}. We used the getPESizes function from the csaw \cite{Lun2016} library to interrogate the distribution of fragment sizes.

Our goal in this analysis was to compare changes in chromosome accessibility across cell-types. This falls within the framework of differential accessibility testing. Here we favored adoption of a window-based method \cite{Lun2014} for detecting differentially accessible regions. Such approaches have previously been implemented for ChIPseq data, notably in the package csaw \cite{Lun2016}.

Reads were counted along the chromosomes in sliding windows of size 200bps. Windows with less than 10 reads were discarded. Further filtering was done by first computing a global background of reads distribution by counting reads in contiguous window size of 1000bps. Composition bias across libraries was alleviated by normalising the libraries using the trimmed mean of M-values (TMM) method \cite{Robinson2010}. Read counts in contiguous windows of 1000bps were again considered for this purpose.

Post normalisation and filtering, the libraries were used for calling differentially accessible windows across cell-types. A paired design ($Y \sim patient + cell type$) was employed to perform a negative binomial regression using functions from the edgeR \cite{Lun2016}. Differentially accessible windows for LSC versus Blast samples and LSC versus pHSC samples were tested for. Post accessibility testing, neighbouring windows were combined to define a region and significance level of the region computed from the p-values of the window-level tests. Multiple testing correction was then done at the region-level using a False Discovery Rate (FDR) cut-off of 0.01. For interpretability of the regions, the number of log-fold change increase (logFC UP) and log-fold change decrease (logFC DOWN) windows it contained were also computed. In addition, the p-value of the best window and the direction of change were also reported. Finally, all regions were annotated to report their distance from neighbouring genes.

\section{Results}

\subsection{LSCs show low expression of TEs.}
Corces et al. \cite{Corces2016} had recently used fluorescent activated cell sorting to isolate leukemic cells from patients with AML. They separated the cells of three distinct stages of AML evolution, pre-leukemic haematopoietic stem cells (pHSCs; $CD34^+CD38^{neg}CD99^{neg}TIM3^{neg}$), leukemic stem cell (LSCs; $CD34^+CD38^{neg}CD99^+TIM3^+$), and leukemic blasts (Blasts; $CD99^+TIM3^+CD45^{mid}SSC^{high}$), characterized their transcriptome, and analysed their coding gene expression patterns \cite{Corces2016}. To investigate the regulation of TEs in the development of AML, we examined the transcriptomes in these stages by measuring the changes in TE expression. When LSCs were compared to pHSCs and Blasts, we identified a significant downregulation of TEs in LSCs (Fig. \ref{fig:tes1} a,b). Among the different classes of TEs, SINE was the most suppressed in LSCs, followed by LTR retrotransposons (Fig. \ref{fig:tes1} a). The most dysregulated TE types in LSCs were Alu, ERV1, ERVL, ERVK, and LTR retrotransposons, all of which showed significant suppression.

We further analysed the dysregulation of TEs in individual AML samples, while tracking the stages of AML. We found that specific TE types were dysregulated, with LSCs showing significant suppression of Alu, ERV3. ERVK, ERVL, and LTR retrotransposons (Fig. \ref{fig:tes1} c). We did not observe significant suppression of LINE1 in LSCs. These results suggested that TEs were dysregulated during AML development, with LSCs showing significant suppression of specific TE types.

\begin{figure}[h!]
\centering
\includegraphics[scale=0.8]{tex/tes/41598_2017_7356_Fig1_HTML.jpg}
\caption{Analysis of differential expression of transposable elements in pre-leukemic stem cells (pHSC), leukemic stem cells (LSC), and Blasts (a) X-axis; Patient identifier.  The expression levels in log10 using the metric transcripts per million (TPM).  The `Transposable Element (TE) Type' classifies individual repeat transcripts into one of 68 unique canonical categories of TEs. Each TE type is contained in one TE Class. (b) Quantiles of the absolute log-fold change of the differentially expressed (DE) TE transcripts in pHSC-LSC, and Blast-LSC samples. Y-axis: absolute log-fold change of each individual DE TE transcript from Fig. \ref{fig:tes1}A. (c) Y-axis: log10 of TPM expression level for each of the 7 paired samples across each clonal point.  The individual patients are denoted with unique colours.}
\label{fig:tes1}
\end{figure}


\subsection{LSCs show suppression of interferon pathways.}
LSCs are known to be resistant to treatment and serve as potential sources of relapse for AML, although the mechanisms behind this resilience are not fully understood \cite{Reinisch2015}. Expression of TEs is known to activate a viral recognition pathway, which causes interferon signalling and immune-mediated cell clearance \cite{Chiappinelli2015, Roulois2015}. Because LSCs showed suppressed TE expression, we investigated whether this TE suppression was associated with the suppression of interferon pathways in LSCs, which could enable its escape from immune-mediated clearance. LSCs showed significantly higher suppression of several Gene Ontology Consortium (GO)-interferon signalling pathways than Blasts (Fig. \ref{fig:tes2}a). When immune-related pathways (with a set of 335 genes, generated by combining 17 canonical immune pathways in MSigDB) and inflammatory pathways (with a set of 649 genes combining acute inflammatory response and inflammatory response in MSigDB and GO) in LSCs and Blasts were compared, LSCs showed significant suppression of the immune-related pathways (Fig. \ref{fig:tes2}b).

However, comparison between LSCs (which showed lower expression of TEs than pHSCs) and pHSCs showed no significant differences in interferon, immune, or inflammatory pathways. This appeared to contradict the model of TE-induced activation of immune pathways. We therefore investigated alternate pathways that could suppress in immune pathways in pHSCs. Interestingly, we found that all pHSCs exhibited very high expression of EVI-1 (pHSCs vs. LSCs, 4.6-fold, $p < 0.0001$; pHSCs vs. Blasts, 3.4-fold, $p < 0.0001$), which is known to suppress immune pathways by downregulating NFκB (a pathway known to be activated by viral RNA) \cite{Xu2012}. Consistent with this finding, we also observed that NF$\kappa$B pathways were more suppressed in pHSCs than Blasts (Blasts and pHSCs showed similar expression of TEs). These findings suggested that both LSCs and pHSCs showed suppression of NF$\kappa$B and immune-related pathways, compared to Blasts. LSCs showed suppressed TE expression and pHSCs showed high expression of EVI-1.

\begin{figure}[h!]
\centering
\includegraphics[height = 15cm, width = 15cm]{tex/tes/41598_2017_7356_Fig2_HTML.jpg}
\caption{Analysis of gene set enrichment for interferon, inflammation, and immune response genes (a) Interferon-related gene sets from GO MSigDB, comparing LSCs and Blasts in AML. (b). Gene set enrichment analysis of combined inflammation and immune gene sets, comparing LSCs and Blasts.  The Bonferroni multiple testing correction significance threshold is denoted as `p.val'. * indicates p \textless 0.025.}
\label{fig:tes2}
\end{figure}

\subsection{Coding gene networks are co-regulated with TEs}
Although TE expression is known to activate immune pathways, the types of TEs that participate in this mechanism are currently unknown. In order to understand the relationship between coding gene expression and the expression of specific TE subtypes, we first performed an unsupervised clustering of the AML samples based on coding gene expression, and found that LSCs formed a well-grouped cluster (Fig. \ref{fig:tes3}). We then analysed the corresponding expression of various TE types and observed a significant suppression in the expression of specific TE types such as Alu, ERV3, ERVK, and LTR ret- rotransposons in LSCs, compared to pHSCs and Blasts (Fig. \ref{fig:tes3}). This suggested that coding gene expression was distinct in samples with low expression of specific types of TEs (LSCs).

Next, in order to investigate which coding gene networks were correlated with specific TE types, we created a genomic association table using the transcriptome from Blasts and LSCs, as shown in Fig. \ref{fig:tes4}. The coding genes were first clustered based on their co-expression to form specific modules. Each module contained unique set of genes that were likely co-regulated and had functional similarities. %For example, module 26 contains many RNA helicase genes (Supplement Fig. 5). 
We correlated these modules to the expression of specific TE types and found that some modules were positively or negatively correlated with the expression of specific TE types. We performed a pathway analysis using the genes in each module for testing the interferon, immune and inflammatory activity, comparing Blasts to LSCs. We identified modules that showed activation (modules 3, 5, 13, 14, 17 and 41) and suppression (22, 24, 26, 29, 30, 39 and 46) of interferon/immune/inflammation gene pathways in Blasts, compared to LSCs (Fig. \ref{fig:tes4}). We then correlated this with the expression of different TE types. As shown in the Fig. \ref{fig:tes4}, the modules that had shown activation of interferon/immune/inflammation genes were positively associated with the expression of specific TE types (Alu, ERVL, ERVK, and LTR retrotransposons) and negatively associated with the expression of ERV1, SAT, and L1. The modules that had shown suppression of the genes in interferon/immune/inflammation were positively associated with the expression of ERV1 and negatively associated with Alu, ERV3, ERVL, and LTR retrotransposons. Chi-square test confirmed a global association between the correlation of positive/negative coding gene module with TE types and the positive/negative enrichment activity of the interferon/immune/inflammation pathways, respectively (p \textless 0.005). This suggested that specific types of TE were significantly linked to interferon/immune/inflammatory pathway activation.

\begin{figure}
    %\centering
    \includegraphics[height = 15cm, width = 15cm]{tex/tes/41598_2017_7356_Fig3_HTML.jpg}
    \caption{Unsupervised hierarchical clustering of coding gene expression in patient samples and the expression levels of the corresponding transposable elements (a)  The image on top depicts the hierarchical clustering of each group (pHSC, LSC, and Blast) based on the average Euclidean distance for the coding gene expression in the patient samples. Below each sample, the expression of the corresponding TE Types (Alu, ERV3, ERVK, ERVL, LTR Retrotransposon, Endogenuous Retroviruses, L1, ERV1, and L2) is shown.  The TE expression is expressed in units of normalized counts per million (CPM) of log10 (1 + CPM).
}
    \label{fig:tes3}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[height = 15cm, width = 15cm]{tex/tes/41598_2017_7356_Fig4_HTML.jpg}
    \caption{Identifying significant associations between the expression of coding gene network and the transposable element types in AML.  The numbers on the Y-axis denote the gene `modules' constructed by identifying gene networks based on co-expression patterns.  The X-axis denotes canonical TE types used for correlating them. The centre  figure of squares represents the correlation matrix for the normalized gene `module' expression and the TE type. * indicates significant associations (p.value \textless 0.05). Each gene `module' was tested for activation of canonical immune and inflammation gene sets in Blasts and compared with LSCs.  The significant (p.value \textless 0.05) pathway activity level for each module is plotted on the left  of Y-axis (yellow indicates significantly higher activation in Blast, and black indicates significantly higher activity in LSCs).}
    \label{fig:tes4}
\end{figure}


\subsection{Pathways that potentially mediate suppression of TEs in LSCs}
The mechanisms behind the regulation of TEs have not been thoroughly investigated. Similar to coding genes, TEs can be regulated both transcrip- tionally and post-transcriptionally. Epigenetic modifications secondary to alterations in ATRX, P53, and SIRT1 and methylation of DNA, have been shown to regulate the expression of TEs \cite{Leonova2013,Elsasser2015,Meter2014}. We investigated whether TEs were suppressed in LSCs through epigenetic mechanisms by analysing its chromatin accessibility using the data from assay for transposase accessible chromatin with high-throughput sequencing (ATAC-seq) for pHSCs, LSCs, and Blasts from Corces et al. \cite{Corces2016}. ATAC-seq has been used for genome-wide mapping of chromatin accessibility. It uses Tn5 transposase to insert sequencing adapters into accessible regions of the chromatin and then uses the sequence reads mapped to the genome to infer accessible regions. Principle component analysis showed that pHSCs were clustered separately from LSCs and Blasts (Fig. 6a). Contrary to our expectations, LSCs, despite having low expression of TEs, had more nucleosome-free regions than pHSCs (Fig. \ref{fig:tes6}b). We analysed the differential accessibility by comparing the accessibility of LSCs to pHSCs, and found 18,099 regions that were significantly more accessible and 441 regions that were significantly less accessible in LSCs compared to pHSCs (Fig. \ref{fig:tes6}b). Comparison of LSCs to Blasts showed no significant differences in the accessible regions. These findings suggested that the suppression of TEs in LSCs was likely not due to increased heterochromatin.

\begin{figure}
    \centering
    \includegraphics[height = 15cm, width = 15cm]{tex/tes/41598_2017_7356_Fig6_HTML.jpg}
    \caption{Chromatin accessibility in pHSCs, LSCs, and Blasts (a) Multi-dimensional scaling plot with two dimensions showing similarity between di erent ATACseq samples: pHSC (blue), LSC (red), and Blast (black). (b) Depicts the differential accessibility using ATACseq sampling data comparing LSCs to pHSC. X-axis is log2 fold change of differentially accessible regions; Y-axis is –log10 of the p.values reported from comparison.  The minimum p.value considered was $5.593e-03$.}
    \label{fig:tes6}
\end{figure}

Because LSCs showed suppressed TE expression despite having more accessible chromatin, we investigated other pathways that could regulate TE expression. A major mechanism for regulating TEs involves their post-transcriptional degradation \cite{Goodier2012,Goodier2013}. We analysed genes known to suppress TEs post-transcriptionally, as described by Goodier et al.\cite{Goodier2016}28, and compared them in LSCs and Blasts. LSCs showed significant upregulation of ATG5 and KIAA0430 (Fig. \ref{fig:tes7}a). Similar to the piRNA system in males, KIAA0430 or meiosis arrest female protein 1 is known to play a key role in repressing TEs during oogenesis \cite{Su2012}. However, its role in regulating TEs in somatic cells has not been reported. Autophagy-related 5 (ATG5), which was significantly upregulated in LSCs, mediated autophagy by enabling the formation of autophagy vesicles. Autophagy is a process by which various intracellular components are transported to the lysosomes and degraded. A recent study showed that autophagy mediates the degradation of TE post-transcriptionally \cite{Guo2014}. Interestingly, LAMP2 was also upregulated in LSCs (Fig. \ref{fig:tes7}b). Recently, it was shown that LAMP2C, a splice isoform of LAMP2, mediated the degradation of RNA via autophagy (RNAutophagy) \cite{Fujiwara2013, Fujiwara2015, Hase2015}. HSP90AA1 (heat shock protein 90 kDa $\alpha$ [cytosolic], class A member 1) is a pathogen receptor that activates autophagy and thus controls the viral infection \cite{Hu2015}. This protein was also seen upregulated in LSCs (Fig. \ref{fig:tes7}a and b).

\begin{figure}
    \centering
    \includegraphics[scale=0.8]{tex/tes/fig7-aml.png}
    \caption{Expression of genes that modulate transposable elements post-transcriptionally (a) Genes that regulate TE post-transcriptionally. Positive fold-change (Y-axis; y \textgreater 0) indicates higher expression in LSCs. Negative fold-change (y \textless 0) indicates higher expression in Blasts. Significant genes are denoted with * p.value \textless 0.05. (b) Autophagy-regulating genes in AML. Expression of LAMP2 and HSP90AA1 pairwise comparison of AML stages. Paired patient measurements are shown with matching colours. Adjusted significance values denoted * p.value \textless 0.05 (c) Expression of RNA helicase genes, DExH genes. Expression of DHX15 in different stages of AML, where paired patient measurements are shown with matching colours. Adjusted significance values denoted * p.value \textless 0.05.}
    \label{fig:tes7}
\end{figure}

RNA helicases are known to bind to and degrade TE post-transcriptionally \cite{Goodier2016,Goodier2012,Bryk2001, Ott2014,Taylor2013, Wu-Scharf2000}. In particular, DHX15 was significantly upregulated in LSCs, compared to Blast (Fig. \ref{fig:tes7}c). These results indicated the possibility that several post-transcriptional mechanisms operated for mediating the suppression of TEs in AML.

\section{Discussion}
Our study is the first to comprehensively evaluate the expression of TEs and its association with coding genes in cancer. We demonstrated that the expression of TEs was dysregulated during the development of AML, with LSCs showing significant suppression. It has been shown that the suppression of the viral recognition pathway conferred resistance to chemotherapy; mutations in MAVS and RIG-1, genes in the viral recognition pathway, have been reported in cancer \cite{Ranoa2016}. We speculated that the expression of TEs could be a potential mechanism for immune-mediated elimination of cancer cells.

Hypomethylating agents have been found to be useful for treating AML, and recent studies have reported that the activation of TEs with the subsequent immune activation was important for their efficacy against cancers \cite{Chiappinelli2015, Roulois2015}. Here, we demonstrated that these mechanisms likely operated naturally during cancer development and progression to enable immune-mediated control of AML. Despite the efficacy of hypomethylating agents against AML, only a minority (~20\%) of patients responded to this therapy \cite{Yun2016}. Among patients who did respond, most eventually developed resistance to therapy with hypomethylating agents. Understanding the regulation of TEs would help us explore predictive factors for hypomethylating treatment and develop novel strategies to prevent relapse in patients treated with hypomethylating agents.

The role of LSCs in the pathogenesis of AML remains controversial. Our results showed that LSCs clearly suppressed the expression of TEs along with distinct coding gene expression. They also showed more suppression of inflammatory pathways, including the NF$\kappa$B pathway. Since Blasts are short lived, they probably did not evolve mechanisms to escape immune-mediated attacks. We speculate that LSCs are a subset of Blasts with the ability to evade immune recognition.

pHSCs, despite having similar expression levels of TEs as Blasts, also showed suppression of inflammatory pathways that prevent the activation of immune signalling. EVI-1, which is known to suppress NF$\kappa$B, was uniquely over-expressed in pHSCs, suggesting that there exists distinct genes which suppress the inflammatory pathways in pHSCs. pHSCs carry mutations in genes regulating the epigenetic machinery and have been clearly demonstrated to precede the development of AML \cite{Jan2012}. pHSCs are resistant to chemotherapy and likely function as reservoirs for relapse of leukaemia \cite{Corces-Zimmerman2014a, Corces-Zimmerman2014b}. High expression of TEs in pHSCs makes them vulnerable to clearance through the viral-recognition pathway; however, this event likely never occurs because of EVI1-mediated suppression of NF$\kappa$B, which is downstream to the viral-recognition pathway. High expression of EVI-1 has been shown to be an indicator of poor risk in AML \cite{Haas2008}. Our analysis is the first to highlight that EVI-1 was significantly expressed at high levels in pHSCs. Targeting EVI-1 in pHSCs could help prevent clonal evolution in AML. For example, miR-133 is known to target EVI-147. It would be important to explore its role in clonal haematopoiesis in the elderly, a condition characterized by expansion of haematopoietic stem cells with mutations in pHSCs.

Our analysis that correlated the expression of coding gene networks to the expression of TE types revealed an association between inflammatory pathways to SINE and LTR families and an anti-association with LINE1. Among the types of TEs, LINE1 is known to have the highest activity of retrotranspositioning and thus has the most potential to cause genomic instability. Hence, LSCs might have co-opted to evolve by suppressing the inflammation-inducing TE classes, while retaining the expression of LINE1, which could potentiate genomic instability and hence clonal evolution.

%We found high expression of several DExH RNA helicases in high-risk MDS. Recent study by Aktas et al. showed that suppression of DHX9 lead to increased levels of Alu48. DHX9 was one of the RNA helicases upregulated in LSCs and high-risk MDS in our study. Targeting DHX9 hence could lead to activation of cancer immunogenicity. Importantly, DHX9 is currently being explored as a target for cancer therapy49–53. RNA helicases bind to single as well as double stranded RNA, and regulate gene splicing. Aberrant splicing events have been reported in patients with MDS, but it is not known whether these splicing factors also regulate TEs. Exploring this function of RNA helicases would enable us to develop drugs targeting them to activate TEs in AML and MDS. In addition to RNA helicases, the role of autophagy in protecting cancer cells from immune attacks via suppressing TE needs to be explored. Drugs targeting autophagy, RNA autophagy (mediated by LAMP2C) in particular, could be promis- ing therapeutic agents against AML and MDS.

Immuno-oncology is emerging as one of the cornerstones of treatment of various cancers. Interferons have long been used in the treatment of cancers, leading to sustained remissions \cite{Talpaz2013, Ortiz2017, Hasselbalch2011}. However, it has been associated with significant systemic toxicities. Activating suppressed TEs, which are known to activate interferons, in cancer cells could potentially accomplish this in a targeted manner.

Our study is the first to show dysregulation of TE in LSCs, revealing its importance in the pathogenesis of AML. Studying direct mechanisms of the regulation of cancer immunosurveillance by TEs in AML could lead to therapies improving long-term survival by manipulating the expression of TEs in leukemic cells.
